### PR TITLE
GSoC: Adding Week 6 - 7 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
image

### CODE REVIEW

File: docs/source/posts/2023/2023-07-08-week-6-praneeth.rst [hunk 1]
```diff
@@ -11,6 +11,9 @@ What did you do this week?
 --------------------------
 This week, I worked on improving the **TextBlock2D** component in the UI system. I started from scratch to address alignment and scaling issues. When resizing the **TextBlock2D**, the text alignment and justification with the background rectangle were inconsistent. To resolve this, I introduced a new "boundingbox" property that calculates the text bounding box based on its content. Additionally, I separated the scaling mode from the resizing action with the new "auto_font_scale" property, enabling automatic font scaling according to the bounding box. This will provide better alignment, justified text, and smoother font scaling for the **TextBlock2D** component. Try it out at `PR #803 <https://github.com/fury-gl/fury/pull/803>`_.
 
+.. image:: https://github.com/fury-gl/fury/assets/64432063/94212105-7259-48da-8fdc-41ee987bda84
+   :align: center
+
```
Comment (JoaoDell) [inline docs/source/posts/2023/2023-07-08-week-6-praneeth.rst:16]: I think alt text is important to be added here for accessibility.

File: docs/source/posts/2023/2023-07-15-week-7-praneeth.rst [hunk 2]
```diff
@@ -11,6 +11,16 @@ What did you do this week?
...
+Text background greater than the actual maximum size:
+
+.. image:: https://github.com/fury-gl/fury/assets/64432063/aaf4a764-4480-4f96-9adf-29d9e28135a6
+   :align: center
+
+Text offset from center: 
+
+.. image:: https://github.com/fury-gl/fury/assets/64432063/0a3bc1e6-a566-4c08-9ca4-a191525b9c97
+   :align: center
+
```
Comment (JoaoDell) [inline docs/source/posts/2023/2023-07-15-week-7-praneeth.rst:23]: Alt text here would be desired as well

**Review by JoaoDell — COMMENTED** [review]
Hello @ganimtron-10, just reviewed your blogposts and everything seems to be building fine. Check some comments I left below.

File: docs/source/posts/2023/2023-07-08-week-6-praneeth.rst (hunk 1 above)
Comment (ganimtron-10) [reply docs/source/posts/2023/2023-07-08-week-6-praneeth.rst:17]: sure will add it!

**Comment by tvcastillod** [comment]
Hello @ganimtron-10 
Good job you have done the last 2 weeks. All is good with the blogposts.

**Review by skoudoro — APPROVED** [review]
A bit short but all good. Thanks @ganimtron-10. Merging!

**Pull request merged**